src/app/components/Navbar/Navbar.tsx

```diff
@@ -30,17 +36,19 @@ export default function Navbar({ title, subtitle, children }: Props) {
               {subtitle || <Skeleton />}
             </div>
           </div>
-          <AccountMenu />
+          <AccountMenu advanced={advanced} />
         </div>
       </div>
       {children && (
         <div>
           <nav className="flex space-x-8">{children}</nav>
         </div>
       )}
-      <div className="md:w-4/12 lg:w-3/12 flex justify-end items-center">
-        <UserMenu />
-      </div>
+      {advanced && (
```

Comment:
Bit confused by the `advance` naming.  
Above `advance` displays the account-switcher/new-account. Here `advance` is also used to show the user-menu.  
Maybe name it like that? `showUserMenu` and `showAccoundSwitcher` or something like that?  
`advance` can be too many things I'm worried.

Reply:
Good suggestion, I agree that advanced can be too vague.